src/sentry/api/endpoints/group_events.py

```diff
@@ -48,7 +50,8 @@ def get(self, request, group):
 
             if query_kwargs['query']:
                 events = events.filter(
-                    message__icontains=query_kwargs['query'],
+                    Q(message__icontains=query_kwargs['query']) | Q(
+                        event_id__iexact=query_kwargs['query'])
```

Comment:
I think we should only attempt to check the `event_id` column when the query matches the right length. We do this elsewhere since we expect the length to be 32: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_group_index.py#L277-L278

I don't think it's a big deal, but might just want to match behavior.

Reply:
The blocker here though is we can't use `__iexact` here. `event_id`s are not case insensitive, and this would cause the query to not use our index since we don't have an index on `UPPER(event_id)` nor is it a `citext` column.

Reply:
@mattrobenolt The id's are case sensitive? That's not behavior I would have expected.